client/src/pages/Messages/Messages.tsx

```diff
@@ -28,6 +32,11 @@ export default function Messages(): JSX.Element {
       setSearch('');
     }
   };
+  useEffect(() => {
```

Comment:
Maybe add this to the socket context? :)

Reply:
Successfully moved this useEffect to socket context. 

commit 5f5ee0e5013612f4bc4fc708bbc8309c80820c4e